### PR TITLE
Resurrected README.md and added old changelog file

### PR DESCRIPTION
**Pull request by elvioaruta**
Using the same README.md with some modifications:
- Added references to colcon instead of catkin
- Added instructions to run the ROS2 package
- Modified ROS words for ROS2
- Modified the geometry_msgs/msg/Twist" example in the readme.md to match the one we have in the example.yaml

### CODE REVIEW

File: inorbit_republisher/CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (adamantivm) [inline inorbit_republisher/CHANGELOG.rst]: I think we should skip this file entirely. I don't think it should get copied from one version of ROS to the other, it should start fresh. In any case, it doesn't matter for what we need it now. Let's not copy it for now.

**Comment by elvioaruta** [comment]
Also, I added /msg/.. to msg types

Old readme.md:
  msg_type: "fruit_msgs/Citrus"
 This one:
  msg_type "fruit_msgs/msg/Citrus"

File: inorbit_republisher/README.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+```yaml
+  republishers:
+  - topic: "/fruits_per_cubic_m"
+    qos: 5
+    msg_type: "fruit_msgs/msg/Citrus"
+    mappings:
+    - field: "num_oranges"
+      mapping_type: "single_field"
+      out:
+        topic: "/inorbit/custom_data/0"
```
Comment (adamantivm) [inline inorbit_republisher/README.md]:
```suggestion
        topic: "/inorbit/custom_data"
```

In ROS 2 the "0" at the end doesn't work. Can you change this in all shown example mappings?

**Review by adamantivm — CHANGES_REQUESTED** [review]
Some updates, then LGTM

**Pull request merged**